### PR TITLE
connection using user and password, for example:: testinfra -v --host…

### PR DESCRIPTION
**Pull request by ftroitino**
…s=sysadmin:sysadminpassword@127.0.0.1 ./test.py

### CODE REVIEW

File: testinfra/backend/paramiko.py [hunk 1]
```diff
@@ -40,6 +40,12 @@ class ParamikoBackend(base.BaseBackend):
 
     def __init__(self, hostspec, ssh_config=None, sudo=False, *args, **kwargs):
         self.host, self.user, self.port = self.parse_hostspec(hostspec)
+        user_pass=self.user.split(":")
+        if len(user_pass) == 2:
+          self.user=user_pass[0]
+          self.password=user_pass[1]
+        else:
+          self.password=Non
```
Comment (daenney) [inline testinfra/backend/paramiko.py]: This is a syntax error, that should be `None`

**Comment by daenney** [comment]
This is lacking integration tests.

**Comment by philpep** [comment]
Hi thanks for the contribution. But I don't see the working part of this PR, this just define a "password" attribute on the class...
So I'm gonna close this PR.

**Comment by philpep** [comment]
I opened #34 for this

**Comment by rmacian** [comment]
So , the code from ptroitino is not valid for the enhacement ?

**Pull request closed without merging**